src/js/components/award/RecipientAddress.jsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * RecipientAddress.jsx
+ * Created by Emily Gullo 01/31/2017
+ **/
+
+import React from 'react';
+
+const propTypes = {
+    recipient: React.PropTypes.object,
+    type: React.PropTypes.string,
+    titleValue: React.PropTypes.string
+};
+
+export default class RecipientAddress extends React.Component {
+
+    render() {
+        const recipient = this.props.recipient;
+        const city = recipient.recipient_city;
+        const stateProvince = recipient.recipient_state_province;
+        let cityState = "";
+
+        if (city && stateProvince) {
+            cityState = `${city}, ${stateProvince}`;
+        }
+        else if (!city && !stateProvince) {
+            cityState = null;
+        }
+        else {
+            cityState = `${city} ${stateProvince}`;
+        }
+        return (
+            <li className={this.props.type}>
+                <div className={`title ${this.props.type}`}>
+                    {this.props.titleValue}
+                </div>
+                <div className="name address">
+                    {recipient.recipient_street}
```

Comment:
It seems from the updated model that this will just return a single concatenated string (without space separation) instead of a multi-line street address when there's more than one of data. For example:

```
Main Building123 First AveUnit 45
```

Instead of
```
Main Building
123 First Ave
Unit 45
```